packages/react-hooks/CHANGELOG.md

```diff
@@ -9,6 +9,10 @@ and adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [1.6.1] - 2020-04-07
 
+### Added
```

Comment:
This should be moved under the "Unreleased", not inside 1.6.1

Reply:
Thanks! It was but due a bad rebase and the new release that happened today, it got mixed up